content/source/docs/cloud/sentinel/manage-policies.html.md

```diff
@@ -33,7 +33,7 @@ Policy sets are managed at an organization level; viewing and modifying them req
 Enforcement levels in Sentinel are used for defining behavior when policies fail to evaluate successfully. Sentinel provides three enforcement modes:
 
 * `hard-mandatory` requires that the policy passes. If a policy fails, the run is halted and may not be applied until the failure is resolved.
-* `soft-mandatory` is much like `hard-mandatory`, but allows an administrator to override policy failures on a case-by-case basis.
+* `soft-mandatory` is much like `hard-mandatory`, but allows any user with the [Manage Policies](../users-teams-organizations/permissions.html#manage-policies) permission to override policy failures on a case-by-case basis.
```

Comment:
It is also applicable to override if they're an organisation owner.

Reply:
@dylanegan An Organization Owner is a persona which is assigned the Manage Policies permission. I wasn't sure if we needed to explicitly call this out or not. Do you think it is necessary?